app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -22,6 +23,10 @@ angular.module("openshiftConsole")
 
         scope.storageClasses = [];
         scope.defaultStorageClass = "";
+        scope.maxNumberClaims = 0;
+        scope.maxStorage = 0;
+        scope.claimCountUsed = 0;
+        scope.claimStorageused = 0;
```

Comment:
i dont see where this variable directly on the scope is getting used, all the refs I see are to either a local var or one that is on scope.storageQuota